migrations/Version20240304230022.php

```diff
@@ -45,15 +45,6 @@ protected function onUp(Schema $schema): void
       FOREIGN KEY (`policy_id`) REFERENCES `policy` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION,
       CONSTRAINT `plan_id_policy_id` UNIQUE (`plan_id`, `policy_id`)
     ) ENGINE='InnoDB' COLLATE 'utf8mb4_unicode_ci';
-    CREATE TABLE `policy_layer` (
```

Comment:
we do not need a layer link to a policy instance.  We already have a link between the plan and the policy and the plan's layers